api/controller/triggers.go

```diff
@@ -113,7 +113,7 @@ func SearchTriggers(database moira.Database, searcher moira.Searcher, page int64
 		searchResults = searchResults[from:to]
 	}
 
-	var triggerIDs []string //nolint
+	triggerIDs := make([]string, 0)
```

Comment:
Here it would be better to set the capacity

Reply:
fixed